clusterloader2/pkg/measurement/common/etcd_metrics.go

```diff
@@ -111,17 +117,29 @@ func (e *etcdMetricsMeasurement) String() string {
 func (e *etcdMetricsMeasurement) startCollecting(host, provider string, interval time.Duration) {
 	e.isRunning = true
 	e.wg.Add(1)
+
+	collectEtcdDatabaseSize := func(host, provider string) error {
```

Comment:
NIT: There is no need for this function to accept the provider string as it doesn't change within the method.

Reply:
i see. I wanted to be as explicit as possible but it makes sense.
I removed the parameters for this method (host as well as provider) since host doesn't change either within the method.